content/en/docs/setup/additional-setup/cni/index.md

```diff
@@ -68,6 +68,7 @@ The following table shows all the options that the `istio-cni` Helm chart suppor
 | `cniConfDir` | | `/etc/cni/net.d` | Must be the same as the environment's `--cni-conf-dir` setting (`kubelet` parameter). |
 | `cniConfFileName` | | | Leave unset to auto-find the first file in the `cni-conf-dir` (as `kubelet` does).  Primarily used for testing `install-cni` plugin configuration.  If set, `install-cni` will inject the plugin configuration into this file in the `cni-conf-dir`. |
 | `psp_cluster_role` | | | This value refers to a `ClusterRole` and can be used to create a `RoleBinding` in the namespace of `istio-cni`. This is useful if you use [Pod Security Policies](https://kubernetes.io/docs/concepts/policy/pod-security-policy) and want to allow `istio-cni` to run as `priviliged` Pods. |
+| `podAnnotations` | | `{}` | Additional custom annotations to be set on pod level |
```

Comment:
```suggestion
| `podAnnotations` | | `{}` | Additional custom annotations to be set on pod level. |
```

Reply:
The default `{}` is confusing. What's the expected format of a value?

Reply:
It's a map of string to string. The default is an empty map.

Reply:
Agree it would be good to say that in the doc here.